src/promptflow/promptflow/_sdk/_service/entry.py

```diff
@@ -131,7 +131,10 @@ def validate_port(port, force_start):
                     f"Please install pywin32 by 'pip install pywin32' and retry. prompt flow "
                     f"service start depends on pywin32.. {ex}"
                 )
-            command = f"waitress-serve --listen=127.0.0.1:{port} promptflow._sdk._service.entry:get_app"
+            command = (
+                f"waitress-serve --listen=127.0.0.1:{port} --threads={PF_SERVICE_WAITRESS_THREADS} "
```

Comment:
Any perf number with this?

Reply:
not yet, Heyi told me that when he runs flow with many traces, the PFS will get stuck; as waitress uses thread, it should be fine with a little more thread worker, so this PR is kind of a workaround for current PFS stuck issue.